### DISCUSSION TITLE
深度终端无法设置回滚行数

### DISCUSSION THREAD

**Discussion by eddiecailinux** [linuxdeepin/developer-center#2859] [Features Request & Ideas | 特性请求 & 头脑风暴]
使用深度终端过程中，当终端中输出了很多很多很多行之后，最开始的行就被覆盖掉了。但是我们开发的时候经常需要查看很久之前的log。设置中没有找到地方可以设置回滚行数。

**Comment by dreamtale90** [comment]
同样问题。并且手动往上翻后，日志一输出就自动回到最新的地方了，不能保持在手动翻的位置。

**Comment by ch593030323** [comment]
> 同样问题。并且手动往上翻后，日志一输出就自动回到最新的地方了，不能保持在手动翻的位置。

更新master的分支,设置中可以看到相关设置
![terminal_setting_scroll](https://user-images.githubusercontent.com/10093841/133426239-f59a603b-234b-4a66-b010-c043e7efc0bb.png)

**Comment by ch593030323** [comment]
> 使用深度终端过程中，当终端中输出了很多很多很多行之后，最开始的行就被覆盖掉了。但是我们开发的时候经常需要查看很久之前的log。设置中没有找到地方可以设置回滚行数。

感谢您的建议，我们会合理考虑此功能

**Comment by dreamtale90** [comment]
> > 同样问题。并且手动往上翻后，日志一输出就自动回到最新的地方了，不能保持在手动翻的位置。
> 
> 更新master的分支,设置中可以看到相关设置
> ![terminal_setting_scroll](https://user-images.githubusercontent.com/10093841/133426239-f59a603b-234b-4a66-b010-c043e7efc0bb.png)

好的，多谢
